Updated README to follow Design Document

mononomori opened the pull request:
- Updated the README to more closely follow the internal design document.
- Added asset to house README images, gifs, videos.

mononomori committed:
feat: updated readme document and added asset folder for readme images and videos

mononomori committed:
fix: fixed screencast video not loading because of improper indentation

mononomori committed:
chore: slight reformatting of document

mononomori committed:
chore: changed imported discrepancies with main to avoid future conflicts

mononomori committed:
chore: converted admin demo to gif because github does not like mp4 apparently...

sbaljeet13 commented on README.md:
```diff
@@ -1,19 +1,164 @@
-# HarmonIQ
 
-Websites such as Whisperify and statsForSpotify utilize the Spotify web API to generate quizzes and display user listening history, but neither of these sites have the ability to add songs to shared playlists. Spotify does offer a feature to create shared playlists but it does not allow the users to vote on the songs that they want to add to the playlists. Even though it offers Spotify Wrapped at ...
```
We should keep this part under the "Competitive Analysis" section

sbaljeet13 commented on README.md:
```diff
@@ -1,19 +1,164 @@
+- _User Logout Process_ (Iteration 1) - A user is done currently using harmonIQ and wishes to logout from their profile.
+	- -> The user clicks the logout button: They are redirected to the harmonIQ login page and can no longer view their profile until the log back in.
+
+## Test Cases:
```
Let's add the Test Cases to a Wiki page in the repo?! It'll be better to build upon them there and prevent the README from keep growing just with the test cases

mononomori replied:
I'm not seeing the wiki header on the repo. Does @ChristianChidiac have to enable it? I've never worked with github wikis before so I might be missing something! Besides that, good idea, do you think it would make sense to split User Stories into a Wiki page as well since that is probably going to grow as well?

ChristianChidiac replied:
I enabled the wikis

sbaljeet13 replied:
Yeah, we can have a page with User Stories in there as well and add the link to both, User Stories and Test Cases, in the README for reference

mononomori committed:
Updated and Reformatted README.md 

Updated the README to follow suggestions in PR: #36 .

Reformatted images  for readability of document.

Changed some headers to more closely follow our Wiki pages.

mononomori committed:
Merge branch 'main' into user-table-iteration-2

mononomori commented:
@sbaljeet13 Added the wiki pages and updated and reformatted theREADME.md. I added Test Case Ids back for the Test Case wiki page just because I think it makes them a bit more searchable. Take a look and let me know what you think.

sbaljeet13 reviewed (approved):
Looks great! Thank you so much for editing and creating the wikis! 🙏🏽
